intermine/webservice.py

```diff
@@ -264,6 +266,8 @@ def __init__(self, root,
         self._list_manager = ListManager(self)
         self.__missing_method_name = None
         if token:
+            if token=="random":
```

Comment:
If I save a list anonymously, then want to view its contents - will this work, or is it a new token every time?

Reply:
It is basically a new token every time a new session is created.

If the session is not quit yet, we can access our saved lists. Once the session is closed, unless we have saved the token in the way which you have mentioned below, we can't get them back.

I haven't tested this feature of reusing tokens, as I thought generating anonymous tokens was just for quick and easy testing purposes which don't need a login every time.

Reply:
okay, that makes sense! I'll give it a look - the main test is whether or not it works well for a jupyter notebook tutorial. So long as it's good for an hour (and not just a single method call) it will be perfect - because you're right, the anon tokens are prettymuch temporary! 👍